api.go

```diff
@@ -211,12 +225,42 @@ type (
 		Status TaskStatus `json:"status"`
 	}
 
+	createTranscodeTaskRequest struct {
```

Comment:
**nit:** I think this should be called `transcodeAssetRequest` instead, since that's the actual API we provide. The task creation is just a side-effect and not the user intent when calling such APIs (which is why the APIs also live under `/asset`). 

Same for the response and actual client functions below, as well as for the "create export task" which IMO should be "export asset".